runtime/tr.source/trj9/build/rules/ztpf-gcc/filetypes.mk

```diff
@@ -0,0 +1,235 @@
+# Copyright (c) 2000, 2017 IBM Corp. and others
```

Comment:
Is 2000 the correct start date?

Reply:
There should only be one copyright date.